src/Avalonia.Base/Media/Fonts/Tables/Metrics/VerticalMetricsTable.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System;
+
+namespace Avalonia.Media.Fonts.Tables.Metrics
+{
+    internal class VerticalMetricsTable
+    {
+        public const string TagName = "vmtx";
+        public static OpenTypeTag Tag { get; } = OpenTypeTag.Parse(TagName);
+
+        private readonly ReadOnlyMemory<byte> _data;
+        private readonly ushort _numOfVMetrics;
+        private readonly int _numGlyphs;
+
+        private VerticalMetricsTable(ReadOnlyMemory<byte> data, ushort numOfVMetrics, int numGlyphs)
+        {
+            _data = data;
+            _numOfVMetrics = numOfVMetrics;
+            _numGlyphs = numGlyphs;
+        }
+
+        public static VerticalMetricsTable? Load(IGlyphTypeface glyphTypeface, ushort numberOfVMetrics, int glyphCount)
```

Comment:
Should probably be named `TryLoad` since there are tables using `Load` throwing if the table isn't found.